includes/Core/Consent_Mode/Consent_Mode.php

```diff
@@ -92,16 +92,28 @@ function () use ( $consent_mode_enabled ) {
 	 * @since 1.122.0
 	 */
 	protected function render_gtag_consent_snippet() {
-		$consent_defaults = array(
-			'ad_personalization' => 'denied',
-			'ad_storage'         => 'denied',
-			'ad_user_data'       => 'denied',
-			'analytics_storage'  => 'denied',
-			// TODO: The value for `region` should be retrieved from $this->consent_mode_settings->get_regions(),
-			// but we'll need to migrate/clean up the incorrect values that were set from the initial release.
-			// See https://github.com/google/site-kit-wp/issues/8444.
-			'region'             => Regions::EU_USER_CONSENT_POLICY,
-			'wait_for_update'    => 500, // Allow 500ms for Consent Management Platforms (CMPs) to update the consent status.
+		/**
```

Comment:
Hi @ankitrox, nice work adding the documentation here for this filter!

However, it has introduced an element of inconsistency, with the `googlesitekit_consent_category_map` filter below it not being similarly documented.

Please can you also add documentation for the `googlesitekit_consent_category_map` filter?